src/index.html

```diff
@@ -1,12 +1,339 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html lang="en"> 
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;0,700;1,500;1,700&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img 
+          src="./images/logo.png" 
+          alt="Moyo logo" 
+          class="logo__image"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphone" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#item" 
+               class="nav__link" 
+               data-qa="nav-hover"
+            >
```

Comment:
Need to check and fix it everywhere
```suggestion
            <a
              href="#item" 
              class="nav__link" 
              data-qa="nav-hover"
            >
```